DSharpPlus/Clients/BaseDiscordClient.cs

```diff
@@ -156,6 +156,7 @@ public async Task<DiscordApplication> GetCurrentApplicationAsync()
         /// Gets a list of regions
         /// </summary>
         /// <returns></returns>
+/// <exception cref="Exceptions.ServerErrorException">Thrown when Discord is unable to process the request.</exception>
```

Comment:
What happened here?

Reply:
The indenting appears to have changed when I mass edited. It looked fine for me in VS, but I can try to change it back here.